core/src/main/java/hudson/model/Queue.java

```diff
@@ -1948,7 +1948,9 @@ default CauseOfBlockage getCauseOfBlockage() {
          * @return false
          *      if the user doesn't have the permission.
          */
-        boolean hasAbortPermission();
+        default boolean hasAbortPermission() {
+            return true;
```

Comment:
Should defaults here check `this instanceof AccessControlled`? Again unlikely to matter in production code which should pick up e.g. https://github.com/jenkinsci/jenkins/blob/6a1b865270c2a1584d79928adf01ab78af8cd63d/core/src/main/java/hudson/model/AbstractProject.java#L1175-L1183

:thinking: check `QueueTaskFilter`